applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -159,8 +159,7 @@ int UPwSmallStrainElement<TDim, TNumNodes>::Check(const ProcessInfo& rCurrentPro
 
     // Base class checks for positive area and Id > 0
     // Verify generic variables
-    int ierr = UPwBaseElement::Check(rCurrentProcessInfo);
-    if (ierr != 0) return ierr;
+    if (auto ierr = UPwBaseElement::Check(rCurrentProcessInfo); ierr != 0) return ierr;
```

Comment:
`ierr` can be made `const`:
```suggestion
    if (const auto ierr = UPwBaseElement::Check(rCurrentProcessInfo); ierr != 0) return ierr;
```

Reply:
Done